docs/source/containerization/overview.mdx

```diff
@@ -1,20 +1,20 @@
 ---
-title: Containerizing the Apollo Router
-subtitle: Run Apollo Router images in containers
-description: Containerize the Apollo Router for portability and scalability. Choose from default or debug images. Deploy in Kubernetes or run in Docker.
+title: Containerizing the GraphOS Router
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)